_posts/2023-01-24-gpu-jupyterlab-in-galaxy.md

```diff
@@ -0,0 +1,61 @@
+
+
+![GPU-enabled JupyterLab in Galaxy Europe for AI](https://raw.githubusercontent.com/anuprulez/gpu_jupyterlab_paper_images/master/jupyterlab_ai.png)
+
+
+### Comparison with Google Colab and Kaggle Kernels
+
+In comparison to popular online infrastructures that provide editors similar to JupyterLab, Galaxy JupyterLab provides free of charge unlimited access to GPU-enabled environments for developing AI models. [Google Colab](https://colab.research.google.com/) and [Kaggle Kernels](https://www.kaggle.com/) provide limited sessions which are as long as only 12 hours. In addition to shorter session time, the amount of memory provided by their sessions is variable and dependent on how much in the past a user has used them. In contrast, Galaxy's JupyterLab provides constant compute resources for unlimited time consisting of 20 GB RAM, 15 GB additional from one GPU, 7 vCPUs and 1 TB of storage for each session of JupyterLab providing a robust environment for developing AI models. In addition, it is possible to create long-running AI model training jobs from any JupyterLab notebook that get executed remotely on Galaxy's cluster. The resulting datasets become available a newly created Galaxy history and the JupyterLab session that started the training job can be safely exited. This feature of training AI models remotely is not available in Google Colab and Kaggle Kernels. Paid subscriptions of Google Colab, pro and pro+, provide a better set of compute resources but they come at a cost of EUR 9.25 and EUR 42.25 per month, respectively.
```

Comment:
its probably not unlimited, isn't it?

Reply:
1 TB? where does this come from?

Reply:
> The resulting datasets become available a newly created Galaxy history

here is something missing

Reply:
> not sure how many days it is? 4 days? We can write several days if not clear

> its probably not unlimited, isn't it?

not sure how many days, 4 it is? we can write several days

Reply:
> 1 TB? where does this come from?

![disk_space](https://user-images.githubusercontent.com/3022518/214347766-beaf0f47-47cd-4d7b-aa76-3cea9b128836.png)

It shows 1TB at `/home/<<docker user>>` . Not sure if it is per session or shared by all users?

Reply:
> Not sure if we need this entire paragraph here, we do not need to compete with them I think.

yes, right